janus_core/calculations/phonons.py

```diff
@@ -44,8 +44,11 @@ class Phonons:  # pylint: disable=too-many-instance-attributes
         Whether to plot various graphs as band stuctures, dos/pdos in svg.
         Default is False.
     symmetrize : bool
-        Whether to symmetrize force constants after cauclation.
+        Whether to symmetrize force constants after calculation.
         Default is False.
+    write_full : bool
+        Whether to maximize information written in various output files.
```

Comment:
Not sure `maximize` is the right word here. 

```suggestion
        Alias to write all output files (hdf5, plot_to_file).
```
?

Reply:
Isn't it just setting what is written out via `auto_band_structure`, not changing which files are written? (That probably could be clearer too)

Reply:
yes at the moment it is just maximizing the info in bands, may be needed for dos/pdos too but I did not check yet.

Reply:
ok refrashed, recheck please.